cloud-config/assembled-templates/20200430-app-es-pg.yml

```diff
@@ -14,7 +14,7 @@ bootcmd:
 - cloud-init-per once aptkeyadd-elasticsearch apt-key add /etc/apt/es_gpg_key
 - cloud-init-per once aptkeyget-postgres wget -qO - https://www.postgresql.org/media/keys/ACCC4CF8.asc > /etc/apt/pg_gpg_key
 - cloud-init-per once aptkeyadd-postgres apt-key add /etc/apt/pg_gpg_key
-- cloud-init-per once curl-nodeinstall curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+- cloud-init-per once curl-nodeinstall curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
```

Comment:
I think you will want to coordinate with @ottojolanki to get Node updated in the AMIs.